myia/grad.py

```diff
@@ -319,7 +319,7 @@ def gen_child(self, g, ng, child):
         with About(child.debug, self.relation):
             self.remap_node((g, child), g, child, ng, ng.apply())
 
-    def gen_fv(self, g, ng, node):
+    def gen_fv_extended(self, g, ng, node):
         """Generate sensitivities for free variables.
 
         Note that the default gen_fv does nothing, so this is different
```

Comment:
Maybe this comment should refer to gen_fv_extended?

Reply:
True